src/panels/lovelace/editor/card-editor/hui-card-picker.ts

```diff
@@ -117,7 +117,7 @@ export class HuiCardPicker extends LitElement {
         </div>
       </div>
       <div class="cards-container">
-        ${(window.customCards || []).map((card) => {
+        ${((window as any).customCards || []).map((card) => {
```

Comment:
Would it be better if you set a const at the top of the file?
```js
const window = window as any;
```

Reply:
We should define a `customCardsWindow` type that types out what we expect to be in it.